rubbery-raccoons/wired/users/views.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib import messages
+from django.contrib.auth import logout
+from django.contrib.auth.forms import UserCreationForm
+from django.shortcuts import redirect, render
+
+
+def register(request):
+    if request.method == "POST":
+        form = UserCreationForm(request.POST)
+        if form.is_valid():
+            form.save()
+            username = form.cleaned_data.get("username")
+            messages.success(request, "Account created for %s." % username)
+            return redirect("login_view")
+        else:
+            messages.error(request, "Account creation failed.")
```

Comment:
As with `Article` creation, I'd like to see better error messages here.

You can access the validation errors from a form: https://docs.djangoproject.com/en/3.1/ref/forms/api/#django.forms.Form.errors